fcrepo-import/src/main/java/org/fcrepo/importer/ArgParser.java

```diff
@@ -26,13 +27,19 @@
 import org.apache.commons.cli.ParseException;
 
 /**
+ * Command-line arguments parser.
+ *
  * @author awoods
+ * @author escowles
  * @since 2016-08-29
  */
 public class ArgParser implements org.fcrepo.importexport.ArgParser {
 
     final private Options cmdOptions;
 
+    public static final String DEFAULT_RDF_EXT = ".ttl";
```

Comment:
I can update the tests to create `Config` objects directly.

But I do think we should rethink the package structure, which seems much too heavy for the number of classes we have now, with each package having a total of 1-2 classes.  But that can be another ticket.